notebooks/py/DARTFISH_Pipeline_-_Human_Occipital_Cortex_-_1_FOV.py

```diff
@@ -225,6 +225,18 @@ def compute_magnitudes(stack, norm_order=2):
 spot_intensities, results = psd.run(zero_norm_stack)
 # EPY: END code
 
+# EPY: START code
+np.unique(spot_intensities['passes_filters'], return_counts=True)
```

Comment:
i'm getting confused between 'filter' -- for image processing and 'filter' -- for annotating rows in an intensity table that meet certain criteria. is there a better name for filter in the latter case? e.g., passes_criteria, passes_thresholds, passes_*?

Reply:
`is_valid` ?